src/utils/__tests__/attachments.test.ts

```
@@ -0,0 +1,184 @@
+    ).toThrow(/MIME/);
+  });
+
+  it("rejects an oversized single attachment using the env override", () => {
+    // 1 KB cap, but content decodes to ~12 bytes — push it past with a long
+    // base64 string that decodes to ~1.5 KB.
+    process.env.MAILTRAP_MAX_ATTACHMENT_SIZE_BYTES = "1024";
+    const oversized = "A".repeat(2400); // decodes to ~1800 bytes
+    expect(() =>
+      validateAndAdaptAttachments([
+        { content: oversized, filename: "big.pdf" },
+      ])
```

ESLint

[error] Replace `⏎········{·content:·oversized,·filename:·"big.pdf"·},⏎······` with `{·content:·oversized,·filename:·"big.pdf"·}`

(prettier/prettier)